.azure-pipelines/packages.yml

```diff
@@ -58,10 +58,12 @@ jobs:
       msbuildArguments: /t:BuildCpp
       maximumCpuCount: true
 
-  - task: DotNetCoreInstaller@0
+  - task: UseDotNet@2
     displayName: install dotnet core sdk
     inputs:
+      packageType: sdk
       version: $(dotnetCoreSdkVersion)
+      includePreviewVersions: true
```

Comment:
Do we really want to use preview version to build deployment packages?

Reply:
I thought I got rid of this after the official 3.0 SDK release...no we do not. I'll remove this

Reply:
Removed in ee4eb0ba384e8578b29e95cf00eed67e7db94f53 . Thanks for finding this!